include/SFML/System/Vector2.inl

```diff
@@ -53,6 +53,138 @@ y(static_cast<T>(vector.y))
 }
 
 
+////////////////////////////////////////////////////////////
+template <typename T>
+T Vector2<T>::length() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::length() is only supported for floating point types");
+
+    using std::sqrt; // allow ADL
+
+    return sqrt(lengthSq());
+}
+
+
+////////////////////////////////////////////////////////////
+template <typename T>
+T Vector2<T>::lengthSq() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::lengthSq() is only supported for floating point types");
+
+    return this->dot(*this);
+}
+
+
+////////////////////////////////////////////////////////////
+template <typename T>
+Vector2<T> Vector2<T>::normalized() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::normalized() is only supported for floating point types");
+
+    assert(*this != Vector2<T>());
+    return (*this) / length();
+}
+
+
+////////////////////////////////////////////////////////////
+template <typename T>
+Angle Vector2<T>::signedAngleTo(const Vector2<T>& rhs) const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::signedAngleTo() is only supported for floating point types");
+
+    using std::atan2; // allow ADL
+
+    assert(*this != Vector2<T>());
+    assert(rhs != Vector2<T>());
+    return radians(atan2(this->cross(rhs), this->dot(rhs)));
+}
+
+
+////////////////////////////////////////////////////////////
+template <typename T>
+Angle Vector2<T>::polarAngle() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::polarAngle() is only supported for floating point types");
+
+    using std::atan2; // allow ADL
+
+    assert(*this != Vector2<T>());
+    return radians(atan2(y, x));
+}
```

Comment:
Would a better name for this be `toAngle` or `asAngle`? That might read easier and better matches other functions that start with verbs.

Reply:
This suggests however that the vector is an angle or representable as angle, like with `Time::asSeconds()` or `Angle::asDegrees()`. But this is not the case.

(btw, "as"/"to" are not verbs 🙂)

Reply:
> (btw, "as"/"to" are not verbs 🙂)

Go easy on me. I've only been speaking and writing English for 27 years.

What about `Vector2<T>::heading`? Heading is defined as the angle relative to X axis. That's already what this implementation is doing.

Reply:
One alternative would also be to just call the method `phi()`.

It's an often-used variable name of the angle in math contexts. And I think in any case, we will need doc to be 100% clear.

Reply:
Yes it's a common term in my professional field of robotics which uses linear algebra and coordinate systems quite often. Heading is defined relative to the X axis. North is sometimes the X axis (see NED coordinate frames) but sometimes East is the X axis (see ENU coordinate frames). Many coordinate frames have no notion of north or east though since some frames are attached to moving vehicles. The only assumption is that we're using a [right handed frame](https://en.wikipedia.org/wiki/Right-hand_rule), which is the conventional way to reason about coordinate frames.

Reply:
Thanks for the input! 

Given my argument about `signedAngleTo()`, we might want to clarify the signed-ness here too.